release/action.yaml

```diff
@@ -0,0 +1,61 @@
+name: Python Release & Publish
+description: Python that publishes a new release.
+inputs:
+  checkout-repo:
+    required: false
+    description: Perform checkout as first step of action
+    default: "true"
+  github-token:
+    required: true
+    description: GitHub token that can checkout the repository as well as create tags/releases against it. e.g. 'secrets.GITHUB_TOKEN'
+    default: ${{ github.token }}
+  pypi-token:
+    required: true
```

Comment:
Is this really required? Not all packages will be released to PyPI.

Reply:
updated to false